tests/utils/test_file.py

```diff
@@ -1,84 +1,53 @@
 import pytest
 import os
 import shutil
-from buildtest.utils.file import (
-    is_dir,
-    is_file,
-    create_file,
-    create_dir,
-    walk_tree,
-)
+import uuid
+from buildtest.utils.file import is_dir, is_file, create_dir, walk_tree, resolve_path
 from buildtest.exceptions import BuildTestError
 
 here = os.path.dirname(os.path.abspath(__file__))
 
 
-@pytest.mark.xfail(
-    reason="Test expected to fail when checking an obscure directory path",
-    raises=BuildTestError,
-)
 def test_checking_directory():
-    is_dir("/xxXXxxyyYYyyYYyyzzZZzzZZzZZZz")
+    dirname = str(uuid.uuid4())
+    assert not is_dir(dirname)
 
 
 @pytest.mark.xfail(
     reason="Test expected to fail when checking an obscure file", raises=BuildTestError
 )
 def test_checking_file():
-    is_file("/xXXXxxXXYyyyYYYyyYzZZZzZZZ")
-
-
-def test_directory_expansion():
-    is_dir("$HOME")
-    is_dir("~")
-
-    assert True is os.path.isdir(os.path.expanduser("~"))
-    assert True is os.path.isdir(os.path.expandvars("$HOME"))
+    file_name = str(uuid.uuid4())
+    assert not is_file(file_name)
 
+    file1 = "~/.profile"
+    file2 = "$HOME/.profile"
```

Comment:
Can we be sure these are universally present?

Reply:
I can't say definitely, but knowing as sysadmin and the Xenial image provides a basic setup but it's not documented about startup configs travis https://docs.travis-ci.com/user/reference/xenial/. I'd say travis user is probably going to be bash, I have not tested this but could make some changes to ``.travis.yml`` for confirmation. 

Since test were working passing it would mean these files exist.

Since the test were working if

Reply:
Did you finish this comment?

Reply:
That was typo in my last comment. Anyhow i didn't touch this regtest. The test will pass on most linux distros if it's a concern we can refactor test if needed. I needed something to validate with user and shell expansion. For user expansion it has to be a file that must exist and startup files are usually found as part of user creation (``useradd``)